Aiko-nfts/aiko-website
# Feature/nav

## Pull Request - User Panel Feature

### Description

This pull request introduces a new `User Panel` feature. The related files are all located under the `components/userpanel` directory. Additionally, a new directory, `assets/userpanel`, has been added to store all of the images used within the `userpanel` components.

### Components included in this PR:

- `bluebutton.tsx`
- `decorhorizontal.tsx`
- `decorvertical.tsx`
- `errorpopup.tsx`
- `jumbostamp.tsx`
- `jumbostampsystem.tsx`
- `nftpanel.tsx`
- `nfts.tsx`
- `stampedition.tsx`
- `stampeditionjumbo.tsx`
- `stampredeem.tsx`
- `stampindividual.tsx`
- `stampreward.tsx`
- `stamprewards.tsx`
- `titlebar.tsx`
- `usernav.tsx`
- `usernavicon.tsx`
- `userprofile.tsx`

### Asset Structure

The `assets` folder now includes a `userpanel` folder, which contains all images used by the components listed above.

### Impact

This PR incorporates the User Panel feature into the main application. Users can now connect their wallet and view the stamps by visiting the new `/userpanel` route. The redeem functionality is currently being developed and will not be included in this PR.

### Test Plan

- Code Review: Please ensure that these changes do not conflict with the existing codebase.
- Check if the components are rendering properly
- Check if all images are loading properly from the `userpanel` assets folder
- Test in different screen sizes to ensure responsive design

### Next Steps

After this PR is merged, the next steps will involve:

- Developing the redeem functionality with an integrated Shopify system: A `server.js` file, integrated with Express.js, will be set up to handle the API for this functionality.
- Enhancements and tweaks within the system, such as music integration, for a more immersive user experience.
